src/init.cpp

```diff
@@ -473,7 +477,7 @@ std::string HelpMessage(HelpMessageMode mode)
 
     strUsage += HelpMessageGroup(_("Node relay options:"));
     if (showDebug)
-        strUsage += HelpMessageOpt("-acceptnonstdtxn", strprintf("Relay and mine \"non-standard\" transactions (%sdefault: %u)", "testnet/regtest only; ", !Params(CBaseChainParams::TESTNET).RequireStandard()));
+        strUsage += HelpMessageOpt("-acceptnonstdtxn", strprintf("Relay and mine \"non-standard\" transactions (%sdefault: %u)", "testnet/regtest only; ", defaultChainParams->RequireStandard()));
```

Comment:
What that text refers to, is that users aren't currently allowed to -acceptnonstdtxn=false on mainnet, they can only set -acceptnonstdtxn=true in testnet/regtest, which makes the fact that the displayed default is true (when in testnet/regtest the default is actually false) even more stupid.
I wanted to have solved this in https://github.com/jtimon/bitcoin/commit/a4b28c491202a1d6c6e77f9439b08b3b2613e818 but it depends on the previously mentioned commit.